File: Pokua/Exercise13.java
import java.util.Random;

public class Exercise13 {
    public static void main(String[] args) {
        Random rand = new Random();
        int toss = rand.nextInt(2);

        if (toss == 0) {
            System.out.println("Heads");
        }else{
            System.out.println("Tails");
        }
    }

    }

File: CODE_CHALLENGES/Beginner_Challenges/Exercise-13.py
#coin toss program
import random
print("Hey there, I am about to make tossing easier for you.")

print("I will toss a coin for you and tell you the result.")
print("Now, lets see what you got!\n")
num = random.choice([0,1])

if num == 0:
    print("Tails")
else:
    print("Heads")